grails-app/views/project/_aboutCitizenScienceProject.gsp

```diff
@@ -119,7 +117,50 @@
             };
 
             map = new MapWithFeatures(mapOptions, mapFeatures);
+        }
+        </g:if>
     }
-</g:if>
+
+    // make sure the list of associated organisations are below the shorter of the two columns.
+    function placeAssociatedOrgs() {
+        console.log("col1 = " + $("#column1").height() + "; col2 = " + $("#column2").height())
+        if ($("#column1").height() > $("#column2").height()) {
+            $("#column2").append($("#associatedOrgs"));
+        } else {
+            $("#column1").append($("#associatedOrgs"));
+        }
     }
+
+    setTimeout(placeAssociatedOrgs, 2000);
 </r:script>
+
+
+
+<script type="text/html" id="associated-orgs">
+<div class="well span12 margin-left-0" id="associatedOrgs">
+    <div class="well-title"><g:message code="project.display.associatedOrgs"/></div>
```

Comment:
Looks like the div will appear even when there is no associated orgs? Is that intentional?
![screen shot 2015-11-05 at 10 27 19 am](https://cloud.githubusercontent.com/assets/4197871/10955436/d59c4e96-83a7-11e5-971f-596a065c079c.png)